test/reporter_output_test.cc

```diff
@@ -712,7 +799,7 @@ ADD_CASES(
 // ========================================================================= //
 // ------------------------- Testing StrEscape JSON ------------------------ //
 // ========================================================================= //
-#if 0 // enable when csv testing code correctly handles multi-line fields
+#if 0  // enable when csv testing code correctly handles multi-line fields
```

Comment:
I didn't notice this change, It have been change by the auto formatter. Should I left it as before?

Reply:
I'd just leave it.